organization/remuneration-policy.md

```diff
@@ -0,0 +1,38 @@
+---
+type: resource
+---
+
+# Remuneration policy
+
+This explains the Foundation for Public Code's approach to remuneration.
+
+## Staff, contractors and interns
+
+Foundation staff salaries are competitive as well as in line with the social mission of the organization.
+
+> Interns may or may not be compensated, depending on whether they're part of a work placement scheme or have an external stipend. We believe strongly that everyone who works for us should receive a living wage. (FACTCHECK?)
+
+The Foundation will reimburse reasonable expenses for staff, contractors and interns.
+
+## Board of directors
+
+> Translate this info: Bestuurders krijgen alleen een vergoeding voor gemaakte onkosten. Ze ontvangen geen vacatiegeld. Gemaakte onkosten zijn kosten die bestuursleden redelijkerwijs hebben door hun rol als bestuurder bij de Foundation.
+
+Members of the Board of directors are not paid for their work as directors. The Foundation will reimburse directors for expenses incurred as a result of their role as director of the Foundation.
+
+> Do directors get different expenses reimbursed than non-directors?
+
+## Volunteers
+
+Volunteers are not paid for their work for the Foundation. We will reimburse reasonable expenses.
+
+## Definition of reasonable expenses
+
+We reimburse reasonable expenses [incurred while working for the Foundation] OR [while carrying out work authorised by the Foundation] for everyone who works with us.  [mits het uitgaven in opdracht van de Foundation betrof].  
```

Comment:
@MirjamvT I'm uncertain of the best translation here. The Dutch original was "mits het uitgaven in opdracht van de Foundation betrof". How precise is 'in opdracht van'? "incurred on behalf of the Foundation?"